requirements/deploy.txt

```diff
@@ -339,7 +339,7 @@ requests==2.31.0
     #   requests-toolbelt
     #   slumber
     #   stripe
-requests-oauthlib==1.4.0
+requests-oauthlib==2.0.0
```

Comment:
I checked this and didn't find anything breaking https://github.com/requests/requests-oauthlib/blob/master/HISTORY.rst#v200-22-march-2024